docs/devops-with-openshift.md

```diff
@@ -0,0 +1,26 @@
+# DevOps with OpenShift
```

Comment:
Minor nit - we have not been using `#` / `<h1>` tags because the title is already in thirty places.

Reply:
@thoraxe updated!